api/v1alpha1/servicebinding_types.go

```diff
@@ -195,3 +202,18 @@ func (sbr ServiceBinding) AsOwnerReference() metav1.OwnerReference {
 		Controller: &ownerRefController,
 	}
 }
+
+func (spec *ServiceBindingSpec) NamingTemplate() string {
+	if spec.BindAsFiles && spec.NamingStrategy == "" {
+		return templates["lowercase"]
+	} else if spec.NamingStrategy == "" {
+		return templates["uppercase"]
+	} else if spec.NamingStrategy != "" {
+		if v, ok := templates[spec.NamingStrategy]; ok {
+			return v
+		} else {
+			return spec.NamingStrategy
+		}
+	}
+	return templates["none"]
```

Comment:
This `return templates["none"]` will never be executed

Reply:
@DhritiShikhar  we still have to return to avoid compilation error.